src/sentry/snuba/events.py

```diff
@@ -26,6 +26,16 @@ class Columns(Enum):
         discover_name="group_id",
         alias="issue.id",
     )
+    # This is needed to query transactions by group id
+    # in the Issue Details page. This will not be
+    # exposed to users through discover search.
```

Comment:
I think we might need to allow discover to filter on this, to support both `open in discover`, and when we switch the `events` tab to use discover.

I think that in general, it's ok to allow us to filter on `group_ids` in discover, but not select it as a column and group by, etc.

Reply:
@wmak is there a reason why we can't expose `group_ids` in discover? want to make sure I don't break anything for discover here 🙏

Reply:
based on the offline chat with Will, `group_ids` hasn't been added to the `discover` dataset but as long we force Discover search to look into the transactions dataset by setting `event.type: transaction` we should be able to search by `group ids`

Reply:
and that should work for `open in discover` and `events` tab 👍